### PR TITLE
Add change your username support

### PR DESCRIPTION
**Pull request by sampoder**

### CODE REVIEW

**Review by lachlanjc — COMMENTED** [review]
Amazing work. Add this command to the help message? https://github.com/hackclub/scrappy/pull/114/files#diff-034feac95900eb32b12b361a61ec5cfeR11

**Comment by MatthewStanciu** [comment]
Awesome, thanks so much! I'm going to merge now and test later and hope it works properly lol

**Comment by sampoder** [comment]
yikes, I tested this and hmmm it was interesting:

I ran `/scrappy-setusername hampoder` the username became `ampode`

Then a couple of minutes later I ran `/scrappy-setusername spampode` the username became `pampod`

So for some reason, the username cuts off the first and last character (I think I knew why). And also the timeout isn't working.

This seems to be because the Airtable field (Last Username Updated Time) isn't being changed

File: pages/api/slack/commands/setusername.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+export default async (req, res) => {
+  if (unverifiedRequest(req))
+    return res.status(400).send('Unverified Slack request!')
+  else res.status(200).end()
+
+  const { text, user_id, response_url } = req.body
+  const args = text.split(' ')
+  let username = args[0] === 'setusername' ? args[1] : args[0]
+  username = username?.substring(1, username.length - 1).replace(' ', '_')
```
Comment (sampoder) [inline pages/api/slack/commands/setusername.js:18]: this line is at fault

**Pull request merged**